### ISSUE TITLE
[mobile] The opened menu is displayed under the grayed out menu options

### ISSUE THREAD

**Issue by rleitan** [mozilla/sumo-experiments#52]
[Environment]:
iOS, Android - Firefox, Safari, Chrome

[Steps to reproduce]:
1.Open https://support.allizom.org/en-US/kb/exp/insecure-password-warning-firefox
2.Tap on one of the nav menu option (e.g. Cookies and cache)

[Expected result]:
The menu should be opened and cover the other menu options ( like on web- see attached photo)

![2019-02-25_13h17_53](https://user-images.githubusercontent.com/41474886/53333862-c5ed7400-38ff-11e9-9391-6d0206030cc3.png)
[Actual result]:
The other menu options are still visible(grayed out) and the opened menu is displayed under grayed out menu options

![image1](https://user-images.githubusercontent.com/41474886/53334029-3bf1db00-3900-11e9-869f-9d7d2b65ab66.png)

**Comment by akatsoulas** [comment]
@rleitan Can you please elaborate why this is a bug? This is happening only when you expand the `Cookies and Cache` submenu and it showcases what's active and what's not. When everything is collapsed the `Insecure password warning in Firefox` is shown with bold.

**Comment by rleitan** [comment]
I agree it's clear what is active and what is not. But I think should be consistent with the desktop behaviour, the inactive menu option (greyed out on mobile) should be overlaid by the active ones, so the menu will  not be displayed in addition to the inactive menu options

**Comment by akatsoulas** [comment]
I would defer this to @madasan as it's more of a UX matter and not an actual bug. We will treat it as such if we want the same behavior as in desktop.
